cypress/e2e/supervision/clients/add-task.cy.js

```diff
@@ -0,0 +1,43 @@
+beforeEach(() => {
+  cy.loginAs("Case Manager");
+  cy.createClient()
+    .withOrder();
+
+  cy.get("@client").then(({id}) => {
+    cy.visit(`/supervision/#/clients/${id}`);
+  });
+});
+
+describe(
+  "Creating self assigned task in supervision",
+  { tags: ["@supervision", "@supervision-regression", "@client-dashboard"] },
+  () => {
+    it("new task is created after a validation error", () => {
+      cy.get('#add-task-button').click()
+      cy.contains("Task type")
+        .closest(".fieldset")
+        .find("select")
+        .select("1");
```

Comment:
It's not super clear what is being clicked here, is there any other attribute you could grab instead? Is there anything like like `[data-role=` or `"input[type=radio][data-core-value=OUTGOING]"`